docs/development.md

```diff
@@ -146,4 +147,29 @@ As an example, in order to build the assets for production, the step `build-prod
    * asset hashing: Additionally, since the site has a [caching layer](https://app.fastly.com) sitting in front of it, 
    we append a hash to the name of the asset in order to invalidate the cache every time we make a release of the site. The configuration 
     is done [here](https://github.com/guardian/support-frontend/blob/master/webpack.config.js#L56). 
- 
\ No newline at end of file
+
+## Available yarn commands
```

Comment:
I would add a new element in the table of content pointing to this new section. The link would be: `6. [Available yarn commands](#available-yarn-commands)`